internal/peer/common/common.go

```diff
@@ -20,6 +21,7 @@ import (
 	pb "github.com/hyperledger/fabric-protos-go/peer"
 	"github.com/hyperledger/fabric/bccsp"
 	"github.com/hyperledger/fabric/bccsp/factory"
+	"github.com/hyperledger/fabric/bccsp/pkcs11"
```

Comment:
This is a common file and not gated by the "pkcs11" build tag.
I believe this solution suffers from the same problem that caused the original solution in #1900 to be reverted. For details see  https://github.com/hyperledger/fabric/pull/1900#issuecomment-713057065.

Reply:
do you want me to remove the `"github.com/hyperledger/fabric/bccsp/pkcs11"` import here ?

Reply:
@denyeart I've fixed it.